docs/Creating-Observables.md

```diff
@@ -3,7 +3,6 @@ This page shows methods that create Observables.
 * [**`just( )`**](http://reactivex.io/documentation/operators/just.html) — convert an object or several objects into an Observable that emits that object or those objects
 * [**`from( )`**](http://reactivex.io/documentation/operators/from.html) — convert an Iterable, a Future, or an Array into an Observable
 * [**`create( )`**](http://reactivex.io/documentation/operators/create.html) — **advanced use only!** create an Observable from scratch by means of a function, consider `fromEmitter` instead
```

Comment:
Please remove the "consider `fromEmitter` instead" from this line as well.

Reply:
It's my mistake. I'm sorry for that.